tests/estimators/classification/test_deeplearning_common.py

```diff
@@ -509,3 +509,94 @@ def test_learning_phase(art_warning, image_dl_estimator):
         assert hasattr(classifier, "_learning_phase")
     except ARTTestException as e:
         art_warning(e)
+
+
+# Note, the following part is for testing the amp tool on pytorch platform only.
+apex_spec = importlib.util.find_spec("apex")
+if apex_spec is not None:
+    amp_spec = importlib.util.find_spec("apex.amp")
+else:
+    amp_spec = None
+amp_found = amp_spec is not None
+
+
+@pytest.mark.skipif(not amp_found, reason="Skip unittests if apex module is not found.")
+@pytest.mark.only_with_platform("pytorch")
+@pytest.mark.parametrize("device_type", ["cpu", "gpu"])
+def test_loss_gradient_amp(
+    get_default_mnist_subset,
+    image_dl_estimator,
+    expected_values_amp,
+    mnist_shape,
+    device_type,
+):
```

Comment:
this test is missing the default `try / except ARTTestException as e: art_warning(e)` at the beginning and end of the test. This is a safeguard which guarantees any ART Meta test Warnings are brought to our attention even if the test passes

Reply:
Solved.